include/private/entry.h

```diff
@@ -27,24 +27,137 @@
 #  include <stumpless/severity.h>
 #  include "private/strbuilder.h"
 
+/**
+ * Frees entry cache
+ *
+ * **Thread Safety: MT-Unsafe**
+ * This function is not thread safe as it destroys resources(entry cache)
+ * that other threads can use.
+ *
+ * **Async Signal Safety: AS-Unsafe**
+ * This function is not safe to call from signal handlers due to destroying
+ * resources(entry cache) that might be used within a function.
+ *
+ * **Async Cancel Safety: AC-Unsafe**
+ * This function is not safe to call from threads that may be asynchronously
+ * cancelled, due to leaving memory in an undefined state.
+ *
+ * @since release v1.6.0
```

Comment:
Have you looked up when this function was added in the commit history? I'm not 100% sure when it came into existence myself. If not, there's no need to do that extra work: you can simply remove the `@since` tag. The tag is only mandatory for new functions moving forward. This of course goes for all of the `@since` tags here.

Reply:
Okay, i wasn't sure about those anyway cause it wasn't clear in the commit history so i am removing those tags in the functions that are not moving forward